src/modules/components/proposal/proposalActions/proposalActions/proposalActions.tsx

```diff
@@ -68,13 +75,28 @@ export const ProposalActions: React.FC<IProposalActionsProps> = (props) => {
                         {...web3Props}
                     />
                 ))}
-                <div className="mt-1 flex w-full flex-col gap-y-3 px-4 pb-4 md:flex-row-reverse md:px-6 md:pb-6">
+                {actions.length === 0 && (
+                    <EmptyState
+                        heading={copy.proposalActionsContainer.empty.heading}
+                        description={emptyStateDescription}
+                        isStacked={false}
+                        objectIllustration={{ object: 'SMART_CONTRACT' }}
+                    />
+                )}
+                <div className="flex md:flex-col">
                     {actions.length > 1 && (
-                        <Button onClick={handleToggleAll} variant="tertiary" size="md" className="shrink-0 md:w-fit">
-                            {expandedItems.length === actions.length
-                                ? copy.proposalActionsContainer.collapse
-                                : copy.proposalActionsContainer.expand}
-                        </Button>
+                        <div className="mt-1 flex w-full flex-col justify-between gap-y-3 px-4 pb-4 md:flex-row md:px-6 md:pb-6">
```

Comment:
Should the children be placed inside this div? Otherwise this div just wraps the button component. We can also probably combine this div with the one above